virt/virt-2-4-release-notes.adoc

```diff
@@ -244,3 +244,6 @@ $ ovirt-aaa-jdbc-tool user unlock admin
 * If a RHV VM disk is in a `Locked` state, you must link:https://access.redhat.com/solutions/396753[unlock the disk] before you can import it.
 
 * `cloud-init` settings are not imported with a RHV virtual machine. You must recreate `cloud-init` after the import process.
+
+// https://bugzilla.redhat.com/show_bug.cgi?id=1880083
+* If you import a VMware VM with UEFI BIOS into OpenShift Virtualization, the VM will not boot. OpenShift Virtualization does not support UEFI. link:https://bugzilla.redhat.com/show_bug.cgi?id=1880083[(*BZ#1880083*)]
```

Comment:
Is it worthwhile starting with 'OpenShift Virtualization does not support UEFI'?
Also can you please 's/OpenShift Virtualization/{VirtProductName}/g'